dispatcher/MongoConnect.py

```diff
@@ -504,8 +504,17 @@ def set_stop_time(self, number, detectors, force):
                                 'max': {'$max': '$rate'}}}
                     ]):
                     rate[doc['_id']] = {'avg': doc['avg'], 'max': doc['max']}
+                channels = set()
+                if 'tpc' in detectors:
+                    # figure out which channels weren't running
+                    readers = list(self.latest_status[detectors]['readers'].keys())
+                    for doc in self.collections['node_status'].find({'host': {'$in': readers}, 'number': int(number)}):
+                        channels |= set(map(int, doc['channels'].keys()))
+                updates = {'rate': rate}
+                if len(channels):
+                    update['no_data_from'] = sorted(list(set(range(494)) - channels))
```

Comment:
My guess is that for high energy it's interesting to see as well (probably also Nveto but that would require you to pull the board map to compute).

Reply:
Probably, but I'd rather avoid including channels where we aren't 100% sure that they're supposed to be contributing. Additionally, if the NV is interested in this feature then they can open a PR to implement it.

Reply:
as you wish, I imagine I'd have found it useful to also check if the HE equivalent did not see any data. But you already showed this once and you'll be the one looking into this most so you know best what info you'd need 😉